media-plugins/distrho-ports/metadata.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
```

Comment:
I think "project" is appropriate here, which is what Gentoo Studio is. A lot of what I implement is because of community input on FGO. (I thought I commented on this, but it seems to have disappeared.)